src/clj/rems/auth/oidc.clj

```diff
@@ -124,7 +119,7 @@
 (defn find-or-create-user! [user-data]
   (let [user (get-user-attributes user-data)
         _ (validate-user! user)
-        user (upsert-user! user)]
+        user (rems.service.users/add-user! user)] ; effectively upserts user
```

Comment:
Could name this function `add-or-update-user!` because it upserts. It would be clear later.